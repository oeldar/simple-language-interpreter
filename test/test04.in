print(1 + 3 = 1 + 3);
print(1 + 3 <> 1 + 3);

print(3 <= 1 + 5);
print(1 + 5 >= 6);

print((5 --4) * 3);
print(4 * -4 * (4 + 3) - 20 - 13);
print(1 * (2 + 1));
