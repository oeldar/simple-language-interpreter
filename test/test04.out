1
0
1
1
27
-145
3
